model-optimizer/extensions/ops/interpolate.py

```diff
@@ -59,6 +60,9 @@ def infer_for_opset4(node: Node):
         for i, axis in enumerate(axes):
             output_shape[axis] = math.floor(scales[i] * output_shape[axis] + 1.0e-5)
 
+    if len([p for p in node.in_ports().values() if not p.disconnected()]) == 4:
```

Comment:
Change to `if node.is_in_port_connected(3):`

Reply:
Fixed.